lib/solr_wrapper/instance.rb

```diff
@@ -103,35 +103,31 @@ def restart
     ##
     # Check the status of a managed Solr service
     def status
-      return true unless config.managed?
-
-      out = exec('status').read
-      out =~ /running on port #{port}/
+      exec('status').read
     rescue
-      false
+      'No status information available'
```

Comment:
It's not really an error message, it's a status message that there is not status info.